lib/vdsm/storage/fileVolume.py

```diff
@@ -583,6 +571,21 @@ def _allocate_volume(cls, vol_path, size, preallocate):
                         log=cls.log):
                     op.run()
 
+    @classmethod
+    def _run_create_image(cls, vol_path, size, format, cow_compat=None):
```

Comment:
In the next patch you create _preallocate_volume helper - for consistency we can name this _create_image.

cow_compat should be qcow2_compat - we want to use the same name for this in the entire project.